kafka_topic_enforcer/README.md

```diff
@@ -63,6 +68,16 @@ Usage: <main class> [options] [command] [command options]
           --unsafemode
             run in unsafe mode, topic deletion is _only_ allowed in this mode
             Default: false
+    strimzi      Generate the Strimzi KafkaTopic resources YAML on stdout from
```

Comment:
nit: formatting seems off here

Reply:
Can you be more specific on what is off? This is directly copied from the command line usage output following the same format as in this file.

Reply:
My bad yeah this formatting is right. I was just comparing to the few lines above but not higher up in the file where the other top-level commands are written